release/air_tests/air_benchmarks/workloads/gpu_batch_prediction.py

```diff
@@ -31,8 +31,16 @@ def preprocess(batch: np.ndarray) -> pd.DataFrame:
 
 @click.command(help="Run Batch prediction on Pytorch ResNet models.")
 @click.option("--data-size-gb", type=int, default=1)
-def main(data_size_gb: int):
-    data_url = f"s3://air-example-data-2/{data_size_gb}G-image-data-synthetic-raw"
+@click.option("--smoke-test", is_flag=True, default=False)
+def main(data_size_gb: int, smoke_test: bool = False):
+    data_url = (
+        f"s3://anonymous@air-example-data-2/{data_size_gb}G-image-data-synthetic-raw"
+    )
+
+    if smoke_test:
+        # Only read one image
+        data_url = [data_url + "/dog.jpg"]
+
     print(f"Running GPU batch prediction with {data_size_gb}GB data from {data_url}")
```

Comment:
nit: Update this since the data size doesn't matter anymore with smoke test?

```suggestion
    size = "one record" if smoke_test else f"{data_size_gb}GB data"
    print(f"Running GPU batch prediction with {size} from {data_url}")
```

Reply:
Actually out of scope of this PR since we're not running this test yet, but we should just rename this test to remove `gpu` from the name since it should work for CPU and GPU